lms/templates/widgets/segment-io-footer.html

```diff
@@ -0,0 +1,14 @@
+% if settings.LMS_SEGMENT_KEY:
+    <!-- begin segment footer -->
```

Comment:
`segment-io.html` is included by `lms/templates/footer.html` as well as `lms/templates/footer-edx-v3.html`, `cms/templates/base.html`, and `lms/templates/main.html`. Have you been able to look into how those templates are used and determine whether it should be removed from any of those?